social-network-api/src/main/java/fs/socialnetworkapi/security/WebConfigSecurity.java

```diff
@@ -0,0 +1,58 @@
+package fs.socialnetworkapi.security;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+
+@Configuration
+@EnableWebSecurity
+@RequiredArgsConstructor
+public class WebConfigSecurity {
+
+  @Value("${jwt.secret}")
+  private String secret;
+  private final JwtFilter jwtFilter;
+  private final String[] publicRoutes = {"/api/v1/registration","/api/v1/login"};
```

Comment:
`private final String[] publicRoutes = {"/api/v1/registration","/api/v1/login"};`
this string not used